docs/api/mount/hasAttribute.md

```diff
@@ -1,17 +1,14 @@
-# hasAttribute(attribute, value)
```

Comment:
Thinking about it, I think hasAttribute should take an optional value. If you can make the changes, please do. Otherwise I'm happy to change myself 😀